napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -94,6 +112,17 @@ def __init__(
         self.setTitleBarWidget(self.title)
         self.visibilityChanged.connect(self._on_visibility_changed)
 
+    @property
+    def shortcut(self):
+        warnings.warn(
+            'The shortcut attribute of QtViewerDockWidget is deprecatedgg '
```

Comment:
This needs to be localized, also we could just use it one place (if it is the same as above) as class variable.

```python
class QtViewerDockWidget(QDockWidget):

    _SHORTCUT_DEPRECATION_STRING = trans._("...")
```

Reply:
typo: `deprecatedgg`